Writerside/topics/Configuring-a-Domain.md

```diff
@@ -0,0 +1,12 @@
+# Configuring a Domain
+
+Catena requires a properly configured domain in order to run in production. This is both for simplicity and to allow us to configure routing to your services 
+across all components of catena.
```

Comment:
General comment for all docs: We should be mindful to use proper capitalization:
- catena -> Catena
- aws -> AWS